docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Introduction to Prometheus
+keywords:
+  - grafana
+  - intro
+  - Prometheus
+  - metrics
+  - time series
+title: What is Prometheus?
+weight: 300
+---
+
+# What is Prometheus?
+
+Observability focuses on understanding the internal state of your systems based on the data they produce, which helps determine if your infrastructure is healthy. Prometheus is a core technology for system observability, but the term "Prometheus" can be confusing because it is used in different contexts. Understanding Prometheus basics, why it’s valuable for system observability, and how people use it in practice will both help you better understand it and help you use Grafana.```
```

Comment:
```suggestion
Observability focuses on understanding the internal state of your systems based on the data they produce, which helps determine if your infrastructure is healthy. Prometheus is a core technology for system observability, but the term "Prometheus" can be confusing because it is used in different contexts. Understanding Prometheus basics, why it’s valuable for system observability, and how people use it in practice will both help you better understand it and help you use Grafana.
```